mysite/code/dataobjects/ApiCallerVersions.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class ApiCallerVersions extends DataObject
```

Comment:
Might be worth adding a PHPDoc to explain what this class is for

Reply:
👍